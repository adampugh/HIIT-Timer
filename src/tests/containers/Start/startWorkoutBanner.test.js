import React from "react";
import { shallow } from "enzyme"; 

import StartWorkoutBanner from "../../../containers/Start/startWorkoutBanner";
import workoutState from "../../fixtures/workouts";

test("should render startWorkoutBanner correctly", () => {
    const wrapper = shallow(<StartWorkoutBanner workout={workoutState.workouts[0].exercises} />);
    expect(wrapper).toMatchSnapshot();
});

test("should render redirect if exercise not passed to startWorkoutBanner", () => {
    const wrapper = shallow(<StartWorkoutBanner workout={[]} />);
    expect(wrapper).toMatchSnapshot();
});