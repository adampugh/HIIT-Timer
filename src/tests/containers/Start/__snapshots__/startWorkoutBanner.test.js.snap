// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render redirect if exercise not passed to startWorkoutBanner 1`] = `
<Redirect
  push={false}
  to="/workouts"
/>
`;

exports[`should render startWorkoutBanner correctly 1`] = `
<div
  className="startWorkoutBanner"
  onClick={[Function]}
  style={
    Object {
      "background": "#40b3e1",
    }
  }
>
  <h1>
    Start
  </h1>
  <h1>
    00
  </h1>
  <div
    className="startWorkoutBanner__grid"
  >
    <div>
      <h3>
        Position
      </h3>
      <h2>
        1
         / 
        2
      </h2>
    </div>
    <div>
      <h3>
        Play
      </h3>
      <h3>
        â–¶
      </h3>
    </div>
    <div>
      <h3>
        Total Time
      </h3>
      <h2>
        00
      </h2>
    </div>
  </div>
  <audio
    id="beep"
    src="beep.mp3"
  />
</div>
`;
